src/store/models/Me.js

```diff
@@ -79,6 +79,7 @@ export default Me
 Me.modelName = 'Me'
 Me.fields = {
   isAdmin: attr(),
+  active: attr(),
```

Comment:
is this actually used for anything?

Reply:
I was thinking it would be needed to reactivate a user, based on the idea that we would ask them to confirm after login whether they actually wanted to be reactivated. But for now we are just automatically reactivating on log-in, so could cut this.